### ISSUE TITLE
K8s v1.19.9 is Out of Support for Azure

### ISSUE THREAD

**Issue by andybouts** [sassoftware/viya4-deployment#168]
K8s v1.19.9 is out of support for Azure; v1.20.9 is their current default, and they have v1.22.2 in preview.

The [prereq](https://github.com/sassoftware/viya4-deployment/blob/main/docs/user/Dependencies.md) page indicates a requirement for 1.19.9.

I seem to be able to get a IAC and Deployment to run with v1.19.11.

The IAC will build with v.1.20.9, but the Deployment will fail:
```bash
TASK [monitoring : cluster-monitoring - deploy] ********************************
fatal: [localhost]: FAILED! => {"changed": true, "cmd": ["/tmp/ansible.guo067c2/viya4-monitoring-kubernetes/monitoring/bin/deploy_monitoring_cluster.sh"], "delta": "0:00:01.368033", "end": "2021-11-17 15:21:42.859281", "msg": "non-zero return code", "rc": 2, "start": "2021-11-17 15:21:41.491248", "stderr": "", "stderr_lines": [], "stdout": "User directory: /tmp/ansible.guo067c2\nHelm client version: 3.7.1\nKubernetes client version: v1.19.9\nKubernetes server version: v1.20.9\n\nnamespace/monitoring created", "stdout_lines": ["User directory: /tmp/ansible.guo067c2", "Helm client version: 3.7.1", "Kubernetes client version: v1.19.9", "Kubernetes server version: v1.20.9", "", "namespace/monitoring created"]}

PLAY RECAP *********************************************************************
localhost                  : ok=43   changed=11   unreachable=0    failed=1    skipped=37   rescued=0    ignored=0
```

**Comment by thpang** [comment]
This is an error the monitoring and logging team are working on that item is in this repo: https://github.com/sassoftware/viya4-monitoring-kubernetes So if you can pass the error along there.

If you have an issue with the v1.19.9 I can fix that. Please adjust the information here. Thx.

**Comment by andybouts** [comment]
I was able to deploy with K8s v1.20.09 by removing the `cluster-monitoring` from the deploy command.  This will work for the interim until the monitoring app will support with a higher K8s version.

PS, I also tested by using K8s v1.19.11, which exceeds the requirements listed in the link above, but is allowed by Azure and it also failed.

Closing issue.

**Comment by andybouts** [comment]
New issue logged, https://github.com/sassoftware/viya4-monitoring-kubernetes/issues/229

**Comment by BryanEllington** [comment]
The monitoring deployment issue has been fixed with our [1.1.1 release](https://github.com/sassoftware/viya4-monitoring-kubernetes/releases/tag/1.1.1).
